docs/en/serial-vault.md

```diff
@@ -0,0 +1,415 @@
+| device-service.url           | The base API URl for the Serial Vault, ending in a trailing slash ("/"). <br> This value must be set to: `https://serial-vault-partners.canonical.com/v1`                                                                                                                                                                                                                                                |   |   |   |
+| device-service.headers       | The API key defined as an HTTP header in JSON string format.  This value must match the one configured for this device model in the SerialVault Admin portal web user interface. <br> You can generate a valid API key with `$pwgen 64 1` and then copy it into the Serial Vault when creating your Model. Or, you can auto create the API key in the Serial Vault when creating a Model and copy it into the `prepare-device` hook script. |   |   |   |
+| registration.proposed-serial | The unique serial number of the device. How to get this serial depends on the device platform.                                                                                                                                                                                                                                                                                                                                         |   |   |   |
+
+## Getting the account-id of your Brand SSO account
+
+Requirements:
+
+ - snapd must be installed.
+ - snapcraft must be installed.
+ - Must be snapcraft logged on as Brand SSO account.
+
+You need to generate a key used by the Serial Vault to sign serial assertions
+and system-user assertions. These keys must be generated under the Brand SSO
+account. After generating a key, you register it, encrypt it, and upload it to
+the Serial Vault. These are all brand activities performed under the Brand SSO
+account. 
+
+Throughout the following tasks, you need to use the account-id of your Brand
+SSO account in various places. You can display this account-id as follows:
+
+1. Login to snapcraft with Brand SSO account credentials:
+
+```bash
+snapcraft login
+```
+
+You will be prompted for the email address and password - be sure to use the email and password of the vendor SSO account.
+
+
+1. Display the SSO account’s account-id:
+```bash
+snapcraft whoami
+```
+
+This should return information in the form: 
+
+```
+email:        vendor.sso@example.com
+developer-id: enAn0ZFfJUhbdHt2T7AjNeCI65vPt2iY
+```
+
+The developer-id listed here is the account-id of the Brand SSO account.
+
+## Providing a key
+
+You need to generate a snapcraft key, register it, encrypt it, and upload it
```

Comment:
Again, kind of awkward to provide a general description of how to do this but not get detailed enough to actually guide the user on how to accomplish these steps. (edit: I see they're described below...I'd mention that they will be described below).